src/ScottPlot5/ScottPlot5/Primitives/FontStyle.cs

```diff
@@ -73,28 +22,6 @@ public override string ToString()
         return $"{Name}, Size {Size}, {Color}";
     }
 
-    private void ClearCachedTypeface()
-    {
-        CachedTypeface = null;
-    }
-
-    public static SKTypeface CreateTypefaceFromName(string font, bool bold, bool italic)
```

Comment:
Unfortunately we need to keep `CreateTypefaceFromName` and `CreateTypefraceFromFile` here for backwards compatibility. `CreateTypefaceFromName` should just forward to `Fonts.GetTypeface`, and `CreateTypefacfeFromFile` is similar, although you'll have to wire it up such that it gets cached properly.

Reply:
That's something I probably disagree with. The `CreateTypefraceFromFile` method as well as the `FontFile` property were added in version **5.0.33 (2 weeks ago)**. It seems to me that while not much time has passed since that moment, we can painlessly break the API. Otherwise, this will lead to duplication of functionality.

There is also one more point to consider: `IFontResolver `is a more universal solution than the existing one. It allows you to set fonts from any source, for example from application resources:
```cs
public class ResFontResolver : IFontResolver
{
    public bool Exists(string fontName)
    {
        return ...
    }

    public SKTypeface? CreateTypeface(string fontName, bool bold, bool italic)
    {
        ...
        using (var asset = AssetLoader.Open(new Uri($"avares://TestApp/Assets/{fontName}.ttf"))) {
            return SKTypeface.FromStream(asset);
        }
    }
}
```

Reply:
I agree that it's unfortunate that we have to preserve a 2 week old API (and one that I don't think really adds much that SkiaSharp doesn't do already). But I think we're stuck with it, the method has shipped and so it should be preserved at least until there's a new major version. The effort of preserving this pales in comparison to the effort of removing it once you consider the support side of things. However, I would recommend annotating it with `[Obsolete]`.

Reply:
See https://github.com/ScottPlot/ScottPlot/pull/3841#issuecomment-2123580075